granulate_utils/linux/mountinfo.py

```diff
@@ -0,0 +1,40 @@
+"""
+See section 3.5 in https://www.kernel.org/doc/Documentation/filesystems/proc.txt
+"""
+from typing import NamedTuple, Iterable, List
+
+
+class Mount(NamedTuple):
+    mount_id: int
+    parent_id: int
+    device: str
+    root: str
+    mount_point: str
+    mount_options: List[str]
+    optional_fields: List[str]
+    filesystem_type: str
+    mount_source: str
+    super_options: List[str]
+
+
+def iter_mountinfo(pid: int) -> Iterable[Mount]:
```

Comment:
Consider defaulting to using `self` if no `pid` is passed

Reply:
Added.